megamek/src/megamek/server/Server.java

```diff
@@ -37272,35 +35915,35 @@ private void handleAttacks(boolean pointblankShot) {
     }
 
     /**
-     * @return
+     * @return the current server instance
      */
     public static Server getServerInstance() {
         return serverInstance;
     }
 
     /**
-     * create a <code>SmokeCloud object and add it to the server list</code>
+     * create a <code>SmokeCloud</code> object and add it to the server list
      *
-     * @param coords
+     * @param coords the location to create the cloud
      */
-    public void createSmoke(Coords coords) {
+    public void createSmoke(Coords coords) { // TODO : Windchild is this needed? No usages
```

Comment:
Do we still need this method given it has zero usages?

Reply:
Unused versions of overloaded methods. They can go.